browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -103,10 +103,6 @@ std::string BraveConfigurator::GetChannel() const {
   return std::string("stable");
 }
 
-std::string BraveConfigurator::GetBrand() const {
```

Comment:
I wonder if we should override `google_brand::GetBrand` or not after this change. Unclear right now to me.